app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -509,3 +510,43 @@ function findConditionalsElseIfUses(content) {
     })
   }
 }
+
+function findEachLoopsWithInvalidIterables(content) {
+  const constants = get(workshopConstants)
+
+  const regex = /@each\s*\(.+in\s+(.+)\s*\)\s*\{/g
+  const constantRegex = /Constant\.([\w\s]*)/
+
+  let match
+  while ((match = regex.exec(content)) != null) {
+    const [eachFull, iterableStr] = match
+
+    if (iterableStr[0] === "[" && iterableStr[iterableStr.length - 1] === "]") {
+      continue
+    }
+
+    const constantMatch = constantRegex.exec(eachFull)
+    if (constantMatch != null) {
+      const [constantFull, constantName] = constantMatch
+
+      if (!constants[constantName]) {
+        const from = match.index + constantMatch.index
+        diagnostics.push({
+          from,
+          to: from + constantFull.length,
+          severity: "error",
+          message: `\"${ constantName }\" is not a known Workshop Constant`
```

Comment:
You don't need to escape the quotes here since it's in a template literal

Reply:
Good point

```suggestion
          message: `"${ constantName }" is not a known Workshop Constant`
```